src/App.tsx

```diff
@@ -1,27 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
+import 'bulma/css/bulma.css';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleAllGoods = () => {
+    getAll()
+      .then(goodsFromServer => {
+        setGoods(goodsFromServer);
+      });
```

Comment:
you repeat this logic 3 times, try to optimize it
you can create one function that will accept needed handler, make a needed request and set data to state

Reply:
Thanks) already done))